### ISSUE TITLE
Pick better accent colours

### ISSUE THREAD

**Issue by craigbox** [istio/istio.io#14666]
We never changed the [callout colours](https://istio.io/latest/docs/releases/contribute/shortcodes/#callouts) when the site was redesigned:

<img width="1261" alt="Screenshot 2024-02-29 at 3 21 13 PM" src="https://github.com/istio/istio.io/assets/132510/dc28f657-1564-40e8-930b-87635798ca48">

Please pick colours that have suitable contrast and better fit the aesthetic of the site.

They appear to be defined in `src/sass/themes/_light_theme.scss`.

```
    --quoteCalloutBackgroundColor: #f8f9fa;
    --quoteCalloutBorderColor: #f8f9fa;
    --tipCalloutBackgroundColor: #f2fff2;
    --tipCalloutBorderColor: #7dc6f2;
    --ideaCalloutBackgroundColor: #ffffe0;
    --ideaCalloutBorderColor: #f7f7d8;
    --warningCalloutBackgroundColor: #fff2f2;
    --warningCalloutBorderColor: #f7eaea;
```

**Comment by Vinujaaa** [comment]
I can pick this up, please assign it to me.

**Comment by craigbox** [comment]
We don't encourage claiming of issues by assignment. Please just send a PR if you have a working implementation.

**Comment by craigbox** [comment]
@Vinujaaa any beautiful colours to offer?

**Comment by ericvn** [comment]
https://github.com/istio/istio.io/issues/12022

**Comment by shruti2522** [comment]
Hi @craigbox, Should we consider changing the text color in the callouts to improve the contrast ratio? Without pastel colors, the current text colors might reduce text visibility.

**Comment by craigbox** [comment]
Sure; treat the boxes holistically from a design and accessibility point of
view.

**Comment by ayushrakesh** [comment]
@craigbox I would like to work on this issue, seems that no-one has completed it.

**Comment by craigbox** [comment]
Fixed by #15062.
